DEFAULT_CONFIG.json5

```diff
@@ -114,6 +114,27 @@
   //      ],
   //  },
 
+  //  /// The declarations downsampling strategy.
+  //  downsampling: {
+  //    items: 
+  //      [
+  //        {
+  //          strategy: "ratelimit",
+  //          // A list of key expressions, only messages matching this expression will be processed, the rest will be passed as is.
+  //          keyexprs: [
+  //            // key_expression
+  //          ],
+  //          // A list of network interfaces messages will be processed on, the rest will be passed as is.
+  //          interfaces: [
+  //            // network interface name
+  //          ],
+  //          // Threshold indicates the minimum interval between messages.
+  //          // All messages arriving with a smaller interval will be skipped
+  //          threshold_ms: 1000,
```

Comment:
I believe configuration should be exposed in a similar way as done in the [ros2dds bridge](https://github.com/eclipse-zenoh/zenoh-plugin-ros2dds/blob/fdf9a1af4f370da05f3a5c13225b296813c00339/DEFAULT_CONFIG.json5#L81), preferring `Hz` versus `time interval`.

From a configuration point of view, I'd prefer to have a set of key-expressions associated to an interface. With the proposed configuration format the same downsampling will be applied to all the interfaces. 

Moreover, with the proposed configuration format the same downsampling will be applied to all key-expressions while it should be specifiable per each key separately.